src/pages/register-a-beacon/application-complete.tsx

```diff
@@ -17,15 +18,15 @@ const ApplicationCompletePage: FunctionComponent = () => (
             </Panel>
             <ApplicationCompleteWhatNext />
             <WarningText>
-              <p className="govuk-body govuk-!-font-weight-bold">
+              <GovUKBody className="govuk-!-font-weight-bold">
```

Comment:
Not for this PR, but worth creating a `<Bold>{ child to which to apply gov-uk-!-font-weight-bold class }</Bold>` component to do this at some point?

Reply:
I like the idea.  How do you see that working?  In this instance, we would want the class `gov-uk-!-font-weight-bold` applied to the `GovUKBody` element so would we have to wrap any existing GovUK React component with the bold class?